test/jobs/update_memberships_job_test.rb

```diff
@@ -41,37 +46,58 @@ def setup
   test 'nested subgroup access level higher' do
     perform_enqueued_jobs do
       assert_equal Member::AccessLevel::GUEST, @group_member.access_level
+      assert_equal Member::AccessLevel::GUEST, @project_member.access_level
       assert_equal Member::AccessLevel::GUEST, @first_subgroup_member.access_level
 
       valid_params = { user: @second_subgroup_member.user, access_level: Member::AccessLevel::MAINTAINER }
       Members::UpdateService.new(@second_subgroup_member, @group_member.namespace, @user, valid_params).execute
 
       assert_equal Member::AccessLevel::GUEST, @group_member.reload.access_level
+      assert_equal Member::AccessLevel::GUEST, @project_member.reload.access_level
       assert_equal Member::AccessLevel::GUEST, @first_subgroup_member.reload.access_level
       assert_equal Member::AccessLevel::MAINTAINER, @second_subgroup_member.reload.access_level
     end
   end
 
+  test 'project access level higher' do
+    assert_equal Member::AccessLevel::GUEST, @group_member.access_level
```

Comment:
Wrap the contents of this test in a `perform_enqueued_jobs` block

Reply:
Please see d672f1edb0420ee57d31351d41d017ef3f6fee43.